fetcher/tar.py

```diff
@@ -99,18 +101,26 @@ def unpack(self, downloadFirst=False):
         update = tarfile.open(name=self.workspacefile, mode='r:*')
```

Comment:
Indeed changes are needed but I don't need a flag for this. I can simply use a stream. Implemented in the last push.